ArchiSteamFarm/Core/AprilFools.cs

```diff
@@ -36,7 +36,7 @@ internal static class AprilFools {
 	internal static void Init(object? state = null) {
 		DateTime now = DateTime.Now;
 
-		if (now is { Month: 4, Day: 1 }) {
+		if (now is (_, 4, 1)) {
```

Comment:
Now this change I actively dislike tbh. Unless Rider shows you the argument name there as it does with parameter names on method calls.

Reply:
It does, and I don't like this change that much either, but I guess .NET added deconstruction for a reason.